poller/index.js

```diff
@@ -27,20 +27,33 @@ const {Spanner} = require('@google-cloud/spanner');
 // GCP service clients
 const metricsClient = new monitoring.MetricServiceClient();
 const pubSub = new PubSub();
-const spannerDefaults = {
+const baseDefaults = {
+  units: 'NODES',
+  scaleOutCoolingMinutes: 5,
+  scaleInCoolingMinutes: 30,
+  scalingMethod: 'STEPWISE'
+};
+const nodesDefaults = {
+  units: 'NODES',
+  minSize: 1,
+  maxSize: 3,
   minNodes: 1,
```

Comment:
I think we should remove the defaults for minNodes/maxNodes.
They are deprecated, so they should only be used in the poller to set minSize/maxSize if they are directly specified in the config, for backward compatibility.